### ISSUE TITLE
Auswirkungen des Brevier der zwölfgöttlichen Unterweisung

### ISSUE THREAD

**Issue by mwberlindeu** [elyukai/optolith-client#744]
Der Brevier der zwölfgöttlichen Unterweisung sollte einen Bonus auf das Talent Götter/ Kulte ermöglichen. Ich habe das mit einem Charakter geprüft, konnte den entsprechenden Bonus aber nicht erkennen.

Ich vermute, dass Boni auf Grund von geeigneten Ausrüstungsgegenständen ggf. noch nicht implementiert sind (Version 1.40)

Es wäre sehr praktisch, wenn derlei Effekte errechnet würden.

Alternativ wäre ein Regelverweis möglich, so dass die entsprechende Regelstelle schnell und ohne langes suchen gefunden werden kann.

**Comment by elyukai** [comment]
Genau, aktuell werden nur die Regeltexte angezeigt, aber nicht angewendet. Auf dem Charakterbogen ist bei aktuellem Layout ja zudem auch kein Platz für die Darstellung von Regeltexten direkt bei den Gegenständen.

**Comment by ZeSandman** [comment]
I not sure it is a good idea to include all equipment bonus directly in the skill checks. 
Equipment can be lost or temporarily unavailable.  players might forget that they don't have their bonus because their book/device isn available/usable right now.

I suggest maybe to add a visual notification/icon on the after the skill name that there is possibly some equipment that might give a bonus, and add such info after the equipment name on it own entry.  Ex: Breviary of the Twelvegods [+1 to Religion checks]

The visual notification could be an indice pointing to the item... ex:

[... on the skill check page]

Religion (#1)      | SGC/SGX/INT |  no  | x 

[... on the Belongings/Equipment page]

(#1) Breviary of the Twelvegods [Religion +1]
